.goreleaser.yml

```
@@ -0,0 +1,116 @@
+    dir: ./src
+    main: .
+    env:
+      - CGO_ENABLED=1
+      - CC=gcc
+    goos:
+      - linux
+    goarch:
+      - amd64
+    ldflags: -s -w
+    binary: "{{ .Os }}-{{ .Arch }}"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)